server/py/framework/db/sqldb/db.py

```diff
@@ -7074,6 +7075,23 @@ def delete_model_endpoints(
             main_table_identifier_values=uids,
         )
 
+    def get_system_id(self, session: Session) -> typing.Optional[str]:
+        system_id_record = (
+            self._query(session, SystemMetadata)
+            .filter(SystemMetadata.key == "system_id")
+            .one_or_none()
+        )
+        if not system_id_record:
+            logger.debug("System id not found in DB")
+            return None
+        return system_id_record.value
+
+    def create_system_id(self, session: Session, system_id: str):
```

Comment:
```suggestion
    def store_system_id(self, session: Session, system_id: str):
    logger.debug("Storing a system id in DB", system_id=system_id)
```
as we already generated(=created) it here.